qutebrowser/config/configdata.yml

```diff
@@ -34,6 +34,9 @@ history_gap_interval:
     Use -1 to disable separation.
 
 ignore_case:
+  renamed: search.ignore_case
```

Comment:
Various tests also use `ignore_case` to test something and now [fail](https://ci.appveyor.com/project/qutebrowser/qutebrowser/build/master-2361/job/sblwxgtsnmu64o1q), so you'll need to adjust those as well.